app/scripts/controllers/tools.js

```diff
@@ -472,6 +492,35 @@ angular.module('oncokbApp')
                                         }
                                     }
                                 });
+                            } else if ($scope.data.evidenceType === 'diagnosticSummary' || $scope.data.evidenceType === 'prognosticSummary') {
```

Comment:
Does it worth to merge these two conditions?

Reply:
Yes, they have same structure. I think we don't need to process them separately.